### PR TITLE
xui add RFI rule in disable list

### PR DESCRIPTION
**Pull request by ernestman28**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by ernestman28** [comment]
FYI. Change request raised for waf rule to be disabled: CHG5001378

File: environments/prod/prod.tfvars [hunk 1]
```diff
@@ -449,6 +449,13 @@ frontends = [
         "942400",
       ]
     }
+    global_exclusions = [
```
Comment (timja) [inline environments/prod/prod.tfvars:452]: needs to be done in the other 3 places too

Comment (andywilkinshmcts) [reply environments/prod/prod.tfvars:452]: Yes - done now

**Comment by andywilkinshmcts** [comment]
global Ignore iss parameter

**Pull request merged**